docs/guides/Create-UWP-Packages-C#.md

```diff
@@ -0,0 +1,217 @@
+
+In this walkthrough you create a NuGet package with a C# UWP component (including a XAML control) that can be used in both Managed and Native projects.
+
+## Prerequisites
+
+1. Visual Studio 2019. Install the 2019 Community edition for free from [visualstudio.com](https://www.visualstudio.com/); you can use the Professional and Enterprise editions as well.
+
+1. NuGet CLI. Download the latest version of `nuget.exe` from [nuget.org/downloads](https://nuget.org/downloads), saving it to a location of your choice (the download is the `.exe` directly). Then add that location to your PATH environment variable if it isn't already.
+
+## Create a UWP Windows Runtime component
+
+1. In Visual Studio, choose **File > New > Project**, search for "uwp c#", select the **Windows Runtime Component (Universal Windows)** template, click next, change the name to ImageEnhancer, and click Create. Accept the default values for Target Version and Minimum Version when prompted.
+
+    ![Creating a new UWP Windows Runtime Component project](media/UWP-NewProject-CS.png)
+
+1. Right click the project in Solution Explorer, select **Add > New Item**, select **Templated Control**, change the name to AwesomeImageControl.cs, and click **Add**:
+
+    ![Adding a new XAML Templated Control item to the project](media/UWP-NewXAMLControl-CS.png)
+
+1. Right-click the project in Solution Explorer and select **Properties.** In the Properties page, choose **Build** tab and enable **XML Documentation File**:
+
+    ![Setting Generate XML Documentation Files to Yes](media/UWP-GenerateXMLDocFiles-CS.png)
+
+1. Right click the *solution* now, select **Batch Build**, check the five build boxes in the dialog as shown below. This makes sure that when you do a build, you generate a full set of artifacts for each of the target systems that Windows supports.
+
+    ![Batch Build](media/UWP-BatchBuild-CS.png)
+
+1. In the Batch Build dialog, and click **Build** to verify the project and create the output files that you need for the NuGet package.
+
+> [!Note]
+> In this walkthrough you use the Debug artifacts for the package. For non-debug package, check the Release options in the Batch Build dialog instead, and refer to the resulting Release folders in the steps that follow.
+
+## Create and update the .nuspec file
+
+To create the initial `.nuspec` file, do the three steps below. The sections that follow then guide you through other necessary updates.
+
+1. Open a command prompt and navigate to the folder containing `ImageEnhancer.csproj` (this will be a subfolder below where the solution file is).
+1. Run the NuGet `spec` command to generate `ImageEnhancer.nuspec` (the name of the file is taken from the name of the `.csroj` file):
```

Comment:
I think adding a link to [nuget spec](https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-spec) command may be helpful.